content/reference/user/artifact_gen/index.md

```diff
@@ -1,33 +1,36 @@
 ---
 title: "Artifact Generation"
-description: "Learn how to generate an artifact for FrontLine from Gatling zip bundle or a maven, SBT or gradle project."
-lead: "Generate an artifact from your Gatling bundle or with a maven, SBT or gradle project."
+description: "Learn how to generate an artifact for Gatling Enterprise from the Gatling zip bundle, or from a Maven, SBT or Gradle project."
+lead: "Generate an artifact from your Gatling bundle or with a Maven, SBT or Gradle project."
 date: 2021-03-08T13:50:32+01:00
 lastmod: 2021-03-08T13:50:32+01:00
 weight: 10040
 ---
 
 ## Generating Artifacts for FrontLine
 
-FrontLine deploys packages containing your compiled Simulations and resources.
+Gatling Enterprise deploys packages containing your compiled Simulations and resources.
 Those packages have to be generated upstream.
-FrontLine is compatible with Gatling 3.3, 3.4 and 3.5.
+Gatling Enterprise is compatible with Gatling 3.3, 3.4, 3.5 and 3.6.
 
 ### Gatling zip bundle
 
-Please copy the [`artifact.sh`](https://raw.githubusercontent.com/gatling/gatling/master/gatling-bundle/src/universal/bin/artifact.sh) or [`artifact.bat`](https://raw.githubusercontent.com/gatling/gatling/master/gatling-bundle/src/universal/bin/artifact.bat) files in the `bin` directory of your Gatling unzipped bundle.
-Those files will be shipped in the bundle in a future official Gatling release.
+Run the script `artifact.sh` (on Linux or macOS) or `artifact.bat` (on Windows), found in the `bin` directory of your unzipped Gatling bundle.
+This will generate a `target/artifact.jar` file. You can then upload this file in the [Artifacts section]({{< ref "../artifact_conf" >}}).
 
-Please run the script matching your operating system and generate the `target/artifact.jar` file.
-You'll have to upload this file in the new [Artifacts section]({{< ref "../artifact_conf" >}}).
+{{< alert warning >}}
+Requires the Gatling bundle version 3.6.0 or superior.
```

Comment:
Hum.
Hard to tell that it requires 3.6.0 when we wrote (L14) that it's compatible from 3.3...

Reply:
This is specifically in the "gatling bundle" section, and it says "gatling bundle" in the sentence. Line 14 is in the general introduction, it includes the other methods of generating a package. Maybe it can be clarified a little bit more though.

The main alternative I see is to add back the old explanation for older versions of Gatling bundle? Something like this:
```
Run the script `artifact.sh` (on Linux or macOS) or `artifact.bat` (on Windows), found in the `bin` directory of your unzipped Gatling bundle.
This will generate a `target/artifact.jar` file. You can then upload this file in the [Artifacts section]({{< ref "../artifact_conf" >}}).

{{< alert warning >}}
The scripts are included in the Gatling bundle version 3.6.0 or superior. For older versions, you will need to copy the
[`artifact.sh`](https://raw.githubusercontent.com/gatling/gatling/master/gatling-bundle/src/universal/bin/artifact.sh)
or [`artifact.bat`](https://raw.githubusercontent.com/gatling/gatling/master/gatling-bundle/src/universal/bin/artifact.bat)
files to the `bin` directory of your unzipped Gatling bundle.
{{< /alert >}}
```

Reply:
Sadly, yep